database/migrations/20211221051741-create_enquiries.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    /**
+     * Add altering commands here.
+     *
+     * Example:
+     * await queryInterface.createTable('users', { id: Sequelize.INTEGER });
+     */
+    await queryInterface.createTable("enquiries", {
+      uuid: {
+        type: Sequelize.UUID,
+        allowNull: false,
+        primaryKey: true,
+        defaultValue: Sequelize.UUIDV4,
+      },
+      user_uuid: {
+        type: Sequelize.UUID,
+        allowNull: false,
+      },
+      type: {
+        type: Sequelize.STRING,
+      },
+      data: Sequelize.JSONB,
+      status: Sequelize.STRING,
+      createdAt: {
+        type: Sequelize.DATE,
+        defaultValue: Sequelize.NOW,
+      },
+      updatedAt: {
+        type: Sequelize.DATE,
+        defaultValue: Sequelize.NOW,
+      },
+      deletedAt: {
+        type: Sequelize.DATE,
+        defaultValue: Sequelize.NOW,
```

Comment:
@rohan099 `deletedAt` column default value should be null, and allowNull should be true